src/components/Footer/Footer.js

```diff
@@ -1,5 +1,136 @@
 import './Footer.scss';
 const Footer = () => {
-  return <div className="footer">footer</div>;
+  return (
+    <div className="footer">
+      <div className="inner">
```

Comment:
```suggestion
      <div className="~~~">
```

좀 더 직관적인 className을 고민해주세요